pkg/minikube/bootstrapper/kubeadm/templates.go

```diff
@@ -39,16 +40,16 @@ etcd:
   dataDir: {{.EtcdDataDir}}
 nodeName: {{.NodeName}}
 {{if .CRISocket}}criSocket: {{.CRISocket}}
-{{end}}{{range .ExtraArgs}}{{.Component}}:{{range $i, $val := printMapInOrder .Options ": " }}
+{{end}}{{range .ExtraArgs}}{{.Component}}ExtraArgs:{{range $i, $val := printMapInOrder .Options ": " }}
```

Comment:
Is this necessary?

Reply:
Yes - it is. We need the raw component names for v1beta1 template, since v1.14 has a sane YAML layout.